app/src/main/java/com/rootstrap/android/ui/MainActivity.kt

```diff
@@ -0,0 +1,40 @@
+package com.rootstrap.android.ui
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.navigation.compose.rememberNavController
+import com.rootstrap.android.ui.compose_navigation.BlackMarketNavigationHost
+import com.rootstrap.android.ui.ui.theme.AndroidBaseTheme
+
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            val navHostController = rememberNavController()
+            SetContentOnSurface {
+                BlackMarketNavigationHost(navHostController = navHostController)
```

Comment:
There will be other `NavigationHost` in the app? I will suggest to remove `BlackMarket` prefix. Nonetheless, everything will be `BlackMarket.....` 😅

Reply:
Good one XD